src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -834,7 +841,22 @@ public int getSize() {
      *             the io exception
      */
     public PagedIterable<GHUser> listCollaborators() throws IOException {
-        return listUsers("collaborators");
+        return listCollaborators(CollaboratorAffiliation.ALL);
```

Comment:
`affiliation` is not a required parameter.  We shouldn't force it to be added. Maybe add a `DEFAULT` to the enum? 

https://docs.github.com/en/free-pro-team@latest/rest/reference/repos#list-repository-collaborators

Reply:
I see your point, I won't force it. I think the enum should map directly to the values in the SDK though, and I'd expect if default behavior is desired, they'd simply call the method with no args. What do you think?

Reply:
> they'd simply call the method with no args

Exactly, but your change forces the addition of the `affiliation` to the API call (line 858 below).  So, your call how you want to do it, but this method call should result in a web api call that doesn't have `affiliation` arg.